tests/Doctrine/Migrations/Tests/Generator/SqlGeneratorTest.php

```diff
@@ -103,7 +103,28 @@ private function prepareGeneratedCode(string $expectedCode) : string
             'SELECT * FROM migrations_table_name',
         ];
 
-        $formattedUpdate = SqlFormatter::format($this->sql[2], false);
+        if (Versions::getVersion('jdorn/sql-formatter') === 'v1.1.0@e3083aafdf447af4abb47342158f6778e03c6d4b') {
```

Comment:
Why target this particular version?

Reply:
It's the version installed when using Composer `--prefer-lowest` option on Travis CI.

Reply:
Shouldn't this work with all the other versions?

Reply:
It should, but having different versions of `jdorn/sql-formatter` means different possible formattings for the SQL queries, so `$expected` must take that into account.

Though instead of checking the version of `jdorn/sql-formatter` I could make `$expected` a regular expression so that it accepts several SQL formattings while still rejecting trailing spaces, WDYT?

Reply:
I replaced the `jdorn/sql-formatter` version check with regular expressions.